modules/grackle/src/main/scala/lucuma/graphql/routes/GrackleGraphQLService.scala

```diff
@@ -53,13 +54,19 @@ class GrackleGraphQLService[F[_]: MonadThrow](
       case Left(errs) => Stream.emit(Left(GrackleException(errs)))
     }
 
-  def format(err: Throwable): Json =
-    QueryInterpreter.mkResponse(None,
-      err match {
-        case GrackleException(ps) => ps
-        case ex                   => List(Problem(ex.getMessage)) // weak, do better here
+  def format(err: Throwable): F[Json] =
+    Logger[F].error(err)("Error computing GraphQL response.")
+      .unlessA(err.isInstanceOf[GrackleException])
```

Comment:
What does `unlessA` do here? I looked at the documentation but am still unclear, esp. being followed by `as`.

Reply:
I'll second the bepuzzlement.

Reply:
I think `unlessA` will run the `F` only if the condition is false. There is also a `whenA` in cats

Reply:
I see, so it filters out `GrackleException`s?

Reply:
So, `GrackleException`s are not being logged here? Is this avoiding double logging of GrackleExceptions, or something like that?

Reply:
If we get a `GrackleException` this means that there were errors in the query itself (typically validation errors) and the errors are encoded as a GraphQL response. So it's a user-level error, not an internal error. Which is to say, it's not really a problem we want to see in the log.